src/functTest/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainerFunctionalTest.groovy

```diff
@@ -118,7 +118,7 @@ class DockerCreateContainerFunctionalTest extends AbstractGroovyDslFunctionalTes
 
         expect:
         def result = build('inspectContainer')
-        result.output.contains("tmpfs /testdata tmpfs rw,nosuid,nodev,noexec,relatime,size=2048k 0 0")
+        result.output.contains("tmpfs /testdata tmpfs rw,nosuid,nodev,noexec,relatime,size=2048k,inode64 0 0")
```

Comment:
Can you explain why this is needed? The test is passing right now on the master branch?

Reply:
It's needed because the output is different, but apparently we'll get different values for it depending on the host system, so it's not a very good fix.

If the order is always the same, then my "future-proofing" idea should work — just truncating it before it gets to the next section of output, basically right after the size — but if the order is not reliable maybe just looking for specific output, like the size, would be better.

I don't know if it listed the specific difference when it failed on the server the first time, but that's the difference I saw when I ran it locally.

Reply:
Honestly, I think checking the console output doesn't make sense and is brittle. The `inspectContainer` task already verifies the relevant configuration.

```
onNext { info ->
    if (info.hostConfig.tmpFs['/testdata'] != 'rw,noexec,nosuid,size=2m') {
        throw new GradleException("'testdata' tmpfs not found")
    }
}
```

I'd say, let's remove the `result.output.contains` line.